opm/simulators/utils/HDF5File.cpp

```diff
@@ -74,23 +76,40 @@ HDF5File::~HDF5File()
 
 void HDF5File::write(const std::string& group,
                      const std::string& dset,
-                     const std::vector<char>& buffer)
+                     const std::vector<char>& buffer) const
 {
     hid_t grp;
     if (groupExists(m_file, group)) {
         grp = H5Gopen2(m_file, group.c_str(), H5P_DEFAULT);
     } else {
+        auto grps = split_string(group, '/');
```

Comment:
This is a bugfix for the write() function, to handle nested groups?

Reply:
well, yes, or, it's not so much that we couldn't write to nested group but to write to group a/b/ you have to create a before you create a/b.